insights/parsers/tests/test_ifcfg.py

```diff
@@ -147,6 +156,8 @@
 
 IFCFG_PATH_NAMED_BOND_MASTER = "etc/sysconfig/network-scripts/ifcfg-eth0"
 
+IFCFG_PATH_NAMED_TEAM_MASTER = "etc/sysconfig/network-scripts/ifcfg-eth0"
```

Comment:
The value of this `IFCFG_PATH_NAMED_TEAM_MASTER` is the same to `IFCFG_PATH_NAMED_BOND_MASTER`. Better just reserve one and rename it to e.g. `IFCFG_PATH_1`